test/test_retry.py

```diff
@@ -11,15 +13,87 @@
 from nameko.testing.services import entrypoint_waiter
 
 from nameko_amqp_retry import Backoff, BackoffPublisher
+from nameko_amqp_retry.backoff import get_backoff_queue_name
+from nameko_amqp_retry.messaging import consume
 from nameko_amqp_retry.rpc import rpc
 
 
+def retry(fn):
+    """ Barebones retry decorator
+    """
+    def wrapper(*a, **kw):
```

Comment:
Oh, this was me being lazy. I normally use `*args, **kwargs` to avoid a single letter variable name. I'll update it before merging.

(This is a standard python thing to accept arbitrary positional and named arguments. See http://stackoverflow.com/questions/36901/what-does-double-star-and-star-do-for-parameters)